editor/src/components/canvas/canvas-strategies/flex-resize-basic-strategy.tsx

```diff
@@ -197,3 +231,36 @@ export function resizeWidthHeight(
     }
   }
 }
+
+const getDimensions = (
+  metadata: ElementInstanceMetadata,
+): {
+  width: number | null
+  height: number | null
+} | null => {
+  const getOffsetPropValue = (
+    name: 'width' | 'height',
+    attrs: PropsOrJSXAttributes,
+  ): number | null => {
+    return foldEither(
+      (_) => null,
+      (v) => (v != null ? v.value : null),
```

Comment:
Minor nitpick but this is just `v?.value ?? null`
I guess it is a matter of taste, but this feels simpler to me.